### ISSUE TITLE
export my own solution?

### ISSUE THREAD

**Issue by mrdulin** [codewars/codewars.com#2161]
### Is your feature request related to a problem? Please describe.

The problem is data security. I am worried that if the codewars database service fails, all my solutions will be lost. I spent a lot of time on codewars and didn't want to waste my time.  So I want to export my own solution, It's kind of a backup. 

#### Describe the solution you'd like

If there is an "export button" on the personal profile, it will be good.

#### Describe alternatives you've considered

I found a tool, it seems like a web crawler, request the page, parse the html, etc...
https://github.com/brunolm/codewars-exporter

#### Additional context

No

**Comment by Blind4Basics** [comment]
(as long as you don't publish them everywhere public... Otherwise, this back up tool is just a bad idea. We already have enough troubbles like that with copycats without making the task even easier...)

**Comment by mrdulin** [comment]
@Blind4Basics yeah, it makes sense. The solutions are published everywhere public is another data security...
